build-package.sh

```diff
@@ -116,15 +116,16 @@ termux_setup_rust() {
 
 # Utility function to setup a current ninja build system.
 termux_setup_ninja() {
-	local NINJA_VERSION=1.8.2
+	local NINJA_VERSION=1.9.0
 	local NINJA_FOLDER=$TERMUX_COMMON_CACHEDIR/ninja-$NINJA_VERSION
 	if [ ! -x "$NINJA_FOLDER/ninja" ]; then
 		mkdir -p "$NINJA_FOLDER"
 		local NINJA_ZIP_FILE=$TERMUX_PKG_TMPDIR/ninja-$NINJA_VERSION.zip
 		termux_download https://github.com/ninja-build/ninja/releases/download/v$NINJA_VERSION/ninja-linux.zip \
 			"$NINJA_ZIP_FILE" \
-			d2fea9ff33b3ef353161ed906f260d565ca55b8ca0568fa07b1d2cab90a84a07
+			609cc10d0f226a4d9050e4d4a57be9ea706858cce64b9132102c3789c868da92
 		unzip "$NINJA_ZIP_FILE" -d "$NINJA_FOLDER"
+		chmod 755 $NINJA_FOLDER/ninja
```

Comment:
For some reason, the default permissions after unzipping were `644`, so it couldn't be executed.